src/ray/gcs/client_def.h

```diff
@@ -0,0 +1,95 @@
+#ifndef RAY_GCS_CLIENT_DEF_H
+#define RAY_GCS_CLIENT_DEF_H
+
+#include <boost/optional/optional.hpp>
+#include <string>
+#include "ray/gcs/tables.h"
+
+namespace ray {
+
+namespace gcs {
+
+/// \class ClientOption
+/// GCS client's options(configuration items), such as service address, service password.
+class ClientOption {
```

Comment:
better rename to `GcsClientOptions` to make its meaning clearer.

Reply:
> better rename to `GcsClientOptions` to make its meaning clearer.
It's too long. There's only one kind of Client here, right?